### ISSUE TITLE
Please make 'develop' the default branch

### ISSUE THREAD

**Issue by bertwesarg** [alpaka-group/llama#27]
@theZiz @psychocoderHPC

**Comment by psychocoderHPC** [comment]
We keep always the stable branch (equal the last release) as default. The reason for that is that a new user should be able to start using what he sees without deeper investigations. 
The pro-developer like you has the advanced task to change the branch for the pull request.

In PIConGPU we avoid using the wrong branch via a CI test https://github.com/ComputationalRadiationPhysics/picongpu/blob/master/test/correctBranchPR

**Comment by bertwesarg** [comment]
But alpaka does not follow this rout, and I took the inspiration from there.

**Comment by psychocoderHPC** [comment]
Yes alpaka is a little bit special in our software zoo. It also has it's own coding style :-(

**Comment by ax3l** [comment]
I have to admit I think as well one does not really need a `stable` branch such as our `master` by default.
We established this for codes that are bare-handedly cloned by users as a default workflow, so they always get a stable version by default. I would be ok to change this as well, e.g. for LLAMA.

**Comment by bernhardmgruber** [comment]
I also would like the default branch to be `develop`. I ran into this issue a few times where I used LLAMA in presentations and told people to grab it from this GitHub repository and they accidentially used `master`, where the features were not merged yet.

I also think people are smart enough when they want to add a dependency to a library to look if there is a last stable version. If they just want to play around, let them use the latest stuff.

@psychocoderHPC further commented, that with making `develop` the default branch, we could also delete the `master` branch.
